Initial Fork of Sheet-Plus

Routable opened the pull request:
Sheet-Plus is a module for Foundry that allows players to use their mobile devices to see their basic inventory details. This is useful, but also has a lot of items lacking, such as spells, stats, and the ability to modify their character sheet.

I would like to take a stab at adding some of this functionality, so I have forked the branch as per the CopyLeft license attributed to it.

Routable commented:
My apologies, I opened a PR under your project as I committed my changes to origin out of habit, and not the forked branch. 

I guess I can take this time to say that the work you have here looks quite promising, but I would like to take your work in a slightly different direction. I like the idea of using a mobile device as a way to view your inventory, available spells, and stat-sheet, but I do not like the idea of being able to interact with the game through the device, as at that point, you're just having a LAN party. 

I'm a big fan of physical figures, rolling, etc, but being able to build your character, and view it while away from the game would be significant to my players, so I'm going to see if I can provide that.

Syrious commented:
Hey there @Routable

well, that might be a happy accident. Reading your intentions on the commit you might want to checkout [Mobile-Companion](https://github.com/Syrious/foundryvtt-mobile-companion) This module is designed to be used on smartphones and will provide inventory for A5e, Dnd5e and Pathfinder and spells coming in the future.

If Mobile-Companion does not suite you needs maybe you want to talk about the features you are missing. I am always open for suggestions :)

Routable commented:
> Hey there @Routable
> 
> well, that might be a happy accident. Reading your intentions on the commit you might want to checkout [Mobile-Companion](https://github.com/Syrious/foundryvtt-mobile-companion) This module is designed to be used on smartphones and will provide inventory for A5e, Dnd5e and Pathfinder and spells coming in the future.
> 
> If Mobile-Companion does not suite you needs maybe you want to talk about the features you are missing. I am always open for suggestions :) 

I've taken a look at Mobile Companion, but I actually prefer the character sheet only, as I believe you put it in some of your code comments - if you offer more than that, it just becomes a LAN party 

I have been cowboy coding some changes directly from my self hosted instance, and there's a few things I've updated:

- I resolved the formatting for the biography text areas for mobile resolutions.

- I added a download character sheet option which currently dumps the players character sheet so they can print it out, or import the data elsewhere.

- I also resolved a few styling issues with mobile screens, so it formats nicely on smaller viewing angles.

I also overwrite the mobile phone screen warnings from foundry, as they are obnoxious.

I'm sure I will think of other things, and will slowly add them over time.
